facturacion-web/src/app/landing/invoicing-page/invoicing-page.component.html

```diff
@@ -3,23 +3,31 @@
     <h1>Nueva Factura</h1>
   </div>
   <hr class="line">
+  <div class="first-line-invoice">
 
-  <div class="form-init">
-    <div>
-      <input id="name" class="input input-client" matInput type="text" placeholder="Nombre de Cliente"
-        autocomplete="off">
+    <div class="blue-div">
+      <div class="blue-label">
+        <span>Nombre del cliente</span>
+      </div>
+      <input class="blue-input-name" matInput placeholder="Ingrese el nombre del Cliente" autocomplete="off">
     </div>
-    <div>
-      <span>Método de Pago:</span>
+
+    <div class="blue-div">
+      <div class="blue-label">
```

Comment:
```suggestion
      <div class="sub-title">
```
en todos los lugares donde se repite